### ISSUE TITLE
Issues with CEA last version (3.11.0)

### ISSUE THREAD

**Issue by BeaHerrero** [architecture-building-systems/CityEnergyAnalyst#2844]
**Describe the bug**
I tried to use it with a group of students and they all got errors when editing the databases and when running the radiance model or importing weather files. The error was "something went wrong" and no more details were shown. As they only difference with mine was the version (I have 3.7.0) I was wondering if there is some problem with the last version released.
**Hardware:**
 - Windows 10
 - CEA version 3.11.0

**Comment by Mar-Ges** [comment]
I tried to download the last version (3.11.0), but i couldn't install it because of a runtime Error:
"R6034
An application has made an attempt to lead the C runtime library incorrectly. 
Please contact the application's support team for more information."

**Comment by daren-thomas** [comment]
@BeaHerrero thank you for reporting this! I'm looking into it...

**Comment by daren-thomas** [comment]
@BeaHerrero I've been trying to replicate your results. It seems importing weather files works for me, so does the radiation script.

When trying to save changes in the Database error, I get the "Something went wrong" error. I'm looking into that now.

**Comment by BeaHerrero** [comment]
We used version 3.7.0 yesterday for doing the same and no one got any
error. Thank you for looking into it.

**Comment by daren-thomas** [comment]
Looks like the error could be here:

```
  File "c:\users\darthoma\documents\cityenergyanalyst\dependencies\python\lib\site-packages\cea\interfaces\dashboard\api\inputs.py", line 351, in get
    return read_all_databases(locator.get_databases_folder())
  File "c:\users\darthoma\documents\cityenergyanalyst\dependencies\python\lib\site-packages\cea\interfaces\dashboard\api\databases.py", line 62, in read_all_databases
    out[category][db_name]['SCHEDULES'][db_file['name']] = schedule_to_dict(db_file['path'])
  File "c:\users\darthoma\documents\cityenergyanalyst\dependencies\python\lib\site-packages\cea\interfaces\dashboard\api\databases.py", line 40, in schedule_to_dict
    schedule_df = schedule_to_dataframe(schedule_path)
  File "c:\users\darthoma\documents\cityenergyanalyst\dependencies\python\lib\site-packages\cea\utilities\schedule_reader.py", line 94, in schedule_to_dataframe
    out['METADATA'] = pd.DataFrame({'metadata': [next(reader)[1]]})
StopIteration
```

I think maybe that code needs to be checked for py3 compatibility.

**Comment by daren-thomas** [comment]
OK. So I found the culprit (in `schedule_reader.py`) - what went wrong (remember, "something went wrong") was that we were writing with a `csv.writer` object to a bytes stream, which is not supported (`open(path, "wb")` was the way to do it in python2, now we just need "w").

I've fixed this and created a PR (#2848). This fix should show up in Release 3.13.
